missingness/missingness.Rmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Arboviral Surveillance and Response Capacity Survey"
+author: "Missingness analysis"
+date: "`r format(Sys.time(), '%Y %B %d')`"
+output: html_document
+---
+
+```{r echo = FALSE, warning = FALSE, message = FALSE}
+
+# load the 2 packages needed:
+library(tidyverse)
+library(flextable)
+
+# process the survey results with our lightly customized version of the WHO script:
+source("survey_375147_R_syntax_file.R")
```

Comment:
I get an error here: 

```
> source("survey_375147_R_syntax_file.R")
Error in file(file, "rt", encoding = fileEncoding) : 
  cannot open the connection
In addition: Warning message:
In file(file, "rt", encoding = fileEncoding) :
  cannot open file 'survey_375147_R_data_file.csv': No such file or directory
```

It seems like there is a csv I should but do not have. Should this be in instructions somewhere?

Reply:
I resolved this by copying the file from the `data` folder to here. But there should be instructions to do this.

Reply:
`instructions.md` explains what to do (or is supposed to...) -- I should have told you to read that first!